src/AutoRest.CSharp/Output/Models/Types/OutputLibrary.cs

```diff
@@ -170,5 +178,97 @@ public RestClient FindRestClient(OperationGroup operationGroup)
             EnsureHeaderModels().TryGetValue(operation, out var model);
             return model;
         }
+
+        private void DecorateOperationGroup()
+        {
+            foreach (var operationsGroup in _codeModel.OperationGroups)
+            {
+                string? resourceType = "";
```

Comment:
why string?

Also for out params you don't need to initialize
```suggestion
                string resourceType;
```

Reply:
because

![image](https://user-images.githubusercontent.com/53361486/108134289-cf32f680-706a-11eb-90ff-a2a32864c304.png)